RPIPs/RPIP-46.md

```diff
@@ -52,38 +52,54 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
     4. `max_node_operator_commission_share_council_adder`: 1%
     5. `allowlisted_controllers`: []
 
-### RPL rewards
-1. The RPL reward system specified in [RPIP-30](RPIP-30.md) SHALL continue
-   1. All staked RPL SHALL be counted for this purpose (both "legacy staked RPL" and "megapool staked RPL" as defined in [RPIP-43](RPIP-43.md))
+### RPL issuance rewards
+- The `proposed_method_share` specified in [RPIP-30](RPIP-30.md) SHALL no longer be used; instead a new `proposed_method_share` SHALL be defined as follows:
```

Comment:
I'm wondering if this is clearer if you just talk about the part of `proposed_method_share` that is supposed to change. Something like. "For <=10% borrowed ETH, `node_weight` is changed from 0 to [...], matching behavior between 10% and 15%".
 
Having it all spelled out is nice for the person implementing it and there is zero chance of any misunderstanding, but most readers care more about what's changing and you kind of have to look at RPIP-30 and compare to get that information.